### DISCUSSION TITLE
Name change of "Aim Assist" mod

### DISCUSSION THREAD

**Discussion by integerrr** [ppy/osu#17375] [Ideas] [answered]
Posting to gauge wider opinion/ gather ideas for new names. Since IMO "aim assist" is not the best name for the mentioned mod as it doesnt actually help your physical aim in anyway but instead moves the hitobject closer to your cursor. Something like "magnetized"(magnetised) suit the behaviour more and makes more sense to me

**Comment by peppy** [comment, marked as answer]
I can agree with this one.

**Reply by peppy** [reply]
@smoogipoo thoughts on this? I think if we're going to change it, doing so sooner is going to be easier.

**Reply by smoogipoo** [reply]
Sure, sounds good

**Reply by peppy** [reply]
#17588 
ppy/osu-web#8762

**Comment by Sublimelime** [comment]
Agree with this as well. Aim assist implies, according to other games' definitions, that the game is gonna move your cursor to the circle (which would likely feel odd, or not work in the case of tablets), not the circle to your cursor. 

Another alternative name could be "Vacuum cursor" or just "Vacuum".
